File: lab_04/hibernate_lab/src/main/java/org/example/Product.java
package org.example;

import jakarta.persistence.*;

@Entity
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int productID;

    private String productName;
    private int unitsInStock;

    @ManyToOne
    private Supplier supplier;

    public Product() {
    }

    public Product(String productName, int unitsInStock) {
        this.productName = productName;
        this.unitsInStock = unitsInStock;
    }
}

File: lab_03/part_02/v_3/Product.cs
namespace lab{
  internal class Product{
    public int ProductID { get; set; }
    public string ProductName { get; set; }
    public int UnitsOnStock { get; set; }
    public Supplier? Supplier { get; set; } = null;
    public override string ToString(){
      return $"{ProductName} ({UnitsOnStock} szt.)";
    }
  }
}